RocketChat/Rocket.Chat
# [FIX] Missing placeholders from encrypted channel preview messages now with expected behavior

## Proposed changes (including videos or screenshots)

Referent to [TC-259]

![Clipboard screenshot](https://user-images.githubusercontent.com/47800334/211094753-30434b6d-46aa-490a-9b27-59770cf0ff24.png)
Note that now the file uses `isE2EEMessage()` function and the `useTranslation` internal hook.



## Issue(s)


## Steps to test or reproduce
In any encrypted channel:

1.     Send a few encrypted messages
2.     Create a new thread
3.     Send messages inside the thread, with the “Send to Channel” option activated.
4.     Reset the user’s E2E password
5.     Log in and enter the Room used in the previous steps without saving the new E2E password.

You'll note that now the preview messages created by the “Send to Channel” option really show up the correct message, instead of its encrypted hash code.

[TC-259]: https://rocketchat.atlassian.net/browse/TC-259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
